lib/src/maintenance.dart

```diff
@@ -0,0 +1,155 @@
+// Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+library pana.maintenance;
+
+import 'dart:async';
+import 'dart:io';
+import 'dart:math';
+
+import 'package:json_annotation/json_annotation.dart';
+import 'package:path/path.dart' as p;
+
+import 'summary.dart' show ToolProblem;
+import 'utils.dart';
+
+part 'maintenance.g.dart';
+
+final Duration _year = const Duration(days: 365);
+final Duration _twoYears = _year * 2;
+
+final List<String> changelogFileNames = const [
+  'changelog.md',
+  'changelog',
+];
+
+final List<String> readmeFileNames = const [
+  'readme.md',
+  'readme',
+];
+
+final List<String> analysisOptionsFiles = const [
+  'analysis_options.yaml',
+  '.analysis_options',
```

Comment:
We should give half credit for `.analysis_options`. Folks should use `analysis_options.yaml`

Reply:
good point, fixing.